docs/common/notification_rules_index.ldoc

```diff
@@ -3,41 +3,41 @@
---   <tr><td><a href='../core_components/notification.html#image'>image</a></td><td>The notification image</td></tr>
---   <tr><td><a href='../core_components/notification.html#images'>images</a></td><td>The notification (animated) images</td></tr>
---   <tr><td><a href='../core_components/notification.html#fg'>fg</a></td><td>Foreground color</td></tr>
---   <tr><td><a href='../core_components/notification.html#bg'>bg</a></td><td>Background color</td></tr>
---   <tr><td><a href='../core_components/notification.html#border_width'>border\_width</a></td><td>Border width</td></tr>
---   <tr><td><a href='../core_components/notification.html#border_color'>border\_color</a></td><td>Border color</td></tr>
---   <tr><td><a href='../core_components/notification.html#shape'>shape</a></td><td>Widget shape</td></tr>
---   <tr><td><a href='../core_components/notification.html#opacity'>opacity</a></td><td>Widget opacity</td></tr>
---   <tr><td><a href='../core_components/notification.html#margin'>margin</a></td><td>Widget margin</td></tr>
---   <tr><td><a href='../core_components/notification.html#preset'>preset</a></td><td>Table with any of the above parameters</td></tr>
---   <tr><td><a href='../core_components/notification.html#callback'>callback</a></td><td>Function that will be called with all arguments</td></tr>
---   <tr><td><a href='../core_components/notification.html#actions'>actions</a></td><td>A table containing strings that represents actions to buttons</td></tr>
---   <tr><td><a href='../core_components/notification.html#ignore'>ignore</a></td><td>Ignore this notification, do not display</td></tr>
---   <tr><td><a href='../core_components/notification.html#suspended'>suspended</a></td><td>Tell if the notification is currently suspended (read only)</td></tr>
---   <tr><td><a href='../core_components/notification.html#is_expired'>is\_expired</a></td><td>If the notification is expired</td></tr>
---   <tr><td><a href='../core_components/notification.html#auto_reset_timeout'>auto\_reset\_timeout</a></td><td>If the timeout needs to be reset when a property changes</td></tr>
---   <tr><td><a href='../core_components/notification.html#ignore_suspend'>ignore\_suspend</a></td><td></td></tr>
---   <tr><td><a href='../core_components/notification.html#clients'>clients</a></td><td>A list of clients associated with this notification</td></tr>
---   <tr><td><a href='../core_components/notification.html#max_width'>max\_width</a></td><td>The maximum popup width</td></tr>
---   <tr><td><a href='../core_components/notification.html#app_name'>app\_name</a></td><td>The application name specified by the notification</td></tr>
---   <tr><td><a href='../core_components/notification.html#widget_template'>widget\_template</a></td><td>The widget template used to represent the notification</td></tr>
+--   <tr><td><a href='../core_components/naughty.notification.html#id'>id</a></td><td>Unique identifier of the notification</td></tr>
```

Comment:
The property linked to doesn't exist. Not sure if that would need to be added, but that would be out of scope in this PR.
So for the time being, I'd just remove the link here.

Reply:
Indeed, same goes for `ignore_suspend` apparently, I hope I didn't miss any more